app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::Base
+
+  # GET /
+  # Fetches a list of articles in descending order by ID.
+  def index
```

Comment:
ApplicationController is a parent of all controllers, and usually not intended to serve requests.
It contains protected or private methods shared to children controllers

Reply:
thanks, fixed